packages/venia-concept/src/components/TextArea/textArea.js

```diff
@@ -13,9 +13,11 @@ export class TextArea extends Component {
             input: string
         }),
         cols: oneOfType([number, string]),
+        field: string.isRequired,
```

Comment:
Turns out that `informed` expects each field to be passed this prop. Found this bug while writing these tests :)

Reply:
Yeah, `field` is required by all Informed fields. It's the key used for storing field state in form context.